packages/cli/package.json

```diff
@@ -4,9 +4,10 @@
   "description": "",
   "main": "dist/bin/pipcook.js",
   "scripts": {
+    "preinstall": "(mkdir -p dist/bin && touch dist/bin/pipcook.js) || ((mkdir dist\\bin || echo 'exists') && echo '' > dist\\bin\\pipcook.js)",
```

Comment:
Do you have a try for https://github.com/charlesguse/run-script-os? This package can split the Windows/Linux script better :)

Reply:
But it does not work for `preinstall` script because the dependencies are not ready at this time.